OrbitCore/PathTest.cpp

```diff
@@ -0,0 +1,18 @@
+#include <gtest/gtest.h>
+
+#include "Path.h"
+
+TEST(FileExists, EmptyFilename) {
+  std::string filename{};
+  EXPECT_FALSE(Path::FileExists(filename));
+}
+
+TEST(FileExists, RootDir) {
+  std::string filename;
+#ifdef _WIN32
+  filename = "C:\\";
+#else
+  filename = "/";
+#endif
+  EXPECT_TRUE(Path::FileExists(filename));
+}
```

Comment:
add a simple check for /dev/null on linux?

Reply:
Good idea, done.